packages/frontend-v2/components/Header/header.tsx

```diff
@@ -0,0 +1,11 @@
+import { Heading, HStack } from "@chakra-ui/react";
+
+export const Logo = (): JSX.Element => {
+  return (
+    <HStack flexDirection="row" alignItems="center" spacing="2px">
+      <Heading size="lg" color="primary.main" fontFamily="logo">Graduate</Heading>
+      <Heading size="xl" color="blue.700" pb="2%">NU</Heading>
+    </HStack>
+  );
+
+}
```

Comment:
Let's rename this to Logo.tsx(capital L) for convention. We're following the convention of naming files based on the primary export of the file.

Reply:
This directory is also a good place to put the HeaderContainer.tsx file. 

Be sure to have a barrel file(index.ts) to export everything in the Header directory and then export everything from the header directory from components/index.ts. 

This allows our imports to be much cleaner - "../components", instead of "../components/Header/HeaderContainer". It also allows us to group all of our imports from the components directory so we can do things like "import {Header, Logo, Button} from "../components". Finally it allows us to reorganize the directory structure within components without having any impact outside components :)

Reply:
Note: If there are any imports from the components directory, be sure to import directly from "../components" instead of the specific component file to reap the benefits of barrels. 

Also we should def not overuse barrels cause it's added boilerplate, but i think they're useful for the components and utils directory since those can get quite nested

Reply:
sounds good :)